windows.ltsc2019/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM mcr.microsoft.com/windows/servercore:ltsc2019
 SHELL ["powershell", "-Command"]
 
 ARG Aws_Cli_Version=2.0.60
+ARG Aws_Iam_Authenticator_Version=0.5.3
```

Comment:
Should we bump the version in the ubuntu dockerfile to 0.5.3 whilst we are here so they align?

Reply:
Yep, all done!